frontend/src/libs/components/dropdown-menu/styles.module.scss

```
@@ -0,0 +1,106 @@
+.dropdown {
+  display: flex;
+  flex-direction: column;
+  align-items: end;
+  width: 50px;
+  background-color: var(--white);
+  border-radius: 10px;
+}
+
+.dropdown-header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 50px;
+  background-color: var(--white);
+  border: 0;
+  cursor: pointer;
+}
+
+.bar {
+  position: relative;
+  z-index: 1;
+  display: block;
+  width: 33px;
+  height: 3px;
+  margin-bottom: 5px;
+  background: var(--gray-200);
+  border-radius: 3px;
+  transform-origin: 4px 0;
+}
+
+.dropdown-body {
+  position: absolute;
+  top: 85px;
+  right: 20px;
+  z-index: 3;
+  display: none;
+  width: 200px;
+  min-width: 160px;
+  margin-top: -3px;
+  overflow: hidden;
+  background-color: var(--white);
+  border-radius: 5px;
+  box-shadow: var(--shadow);
+}
+
+.open {
+  display: block;
+}
+
+.dropdown-item {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding: 10px;
+  background-color: var(--white);
+  border: 0;
+}
+
+.item {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-self: baseline;
+  gap: 5px;
+}
+
+hr {
+  width: 100%;
+  margin: 0;
+  border: 0.5px solid var(--gray-200);
+}
+
+hr:last-child {
+  border: 0;
+}
+
+.dropdown-item:hover {
+  background-color: var(--gray-100);
+  cursor: pointer;
+}
+
+.icon {
+  width: 24px;
+  height: 24px;
+  filter: invert(45%) sepia(7%) saturate(555%) hue-rotate(185deg)
+    brightness(93%) contrast(86%);
+}
+
+.icon-selected {
+  filter: invert(43%) sepia(94%) saturate(426%) hue-rotate(185deg)
+    brightness(94%) contrast(97%);
+}
+
+.title {
+  display: flex;
+  justify-content: baseline;
+  align-self: last baseline;
+}
+
+@media (width > 480px) {
```

let's make all queries in the same style
```suggestion
@media (width < 480px) {
```